find.lua

```diff
@@ -0,0 +1,461 @@
+local ffi = require 'ffi'
+
+find = {}
+find.__index = find
+
+local function call(layer, f, ...)
+   local status = cudnn.call(f, ...)
+   if status ~= ffi.C.CUDNN_STATUS_SUCCESS and cudnn.verbose then
+      local pad = ffi.new('int[5]')
```

Comment:
That was fallback debug left in - I copy-pasted block from the fallback and only printed out mode to make sure the mode flip occured. Not too useful in general. Removed.